debian/tests/control

```diff
@@ -1,10 +1,31 @@
 # DEP-8 continuous integration tests.
 #
-#  TODO: Add in tree unit testing whenever it becomes available.
+#  TODO: Make integration testing run
 #
 
 Test-Command: nicotine --version | grep Nicotine+
 Depends: nicotine
 Restrictions: isolation-container
 Features: test-name=test-version
 
+Test-Command: pytest-3 tests/unit
+Depends:
+    nicotine,
+    python3-pytest
+Restrictions: isolation-container
+Features: test-name=unit-tests
+
+Test-Command: xvfb python3 tests/integration/run_app.py
+Depends:
+    nicotine,
+    python3-dogtail,
+    python3-pyatspi,
+    python3-cairo,
```

Comment:
Are all of these explicit dependencies actually necessary for the test? You can safely remove any that `nicotine` itself depends on.

Reply:
Yes, this list can be much shorter actually.